web/src/features/weather-layers/solar/SolarLayer.tsx

```diff
@@ -93,7 +93,7 @@ export default function SolarLayer({ map }: { map?: maplibregl.Map }) {
         map.removeSource('solar');
       }
     };
-  }, [map, node, isVisibleReference.current]);
```

Comment:
I have added it back but ESlint throws a warning now:
```
React Hook useEffect has an unnecessary dependency: 'isVisibleReference.current'. Either exclude it or remove the dependency array. Mutable values like 'isVisibleReference.current' aren't valid dependencies because mutating them doesn't re-render the component.
```
However since it's just a warning we can adress it later.